### DISCUSSION TITLE
Archive Requests/Cleanup

### DISCUSSION THREAD

**Discussion by th30p3rat0r** [sct/overseerr#2220] [Feature Requests]
Submitted this on the discord, but was hoping a feature for Archiving requests instead of having to just delete them would be possible/has been thought about?  Of course we can just filter the requests queue by pending, but it would be nice to be able to keep a clean request view without the filter specifically.  Cheers!

**Comment by SunsparcSolaris** [comment]
A bulk delete/archive would be nice as well. Instead of having to delete each request individually, allow a filter similar to how Radarr/Sonarr does it where you can only show Available/Requested and take action from there.

I specifically go through the Available's every so often and have to individually clean them up, a feature to only show those and select all -> delete would be nice.

**Comment by BloodBlight** [comment]
Even just a check box system to delete multiple at a time would be a very welcomed addition!
